.travis.yml

```diff
@@ -8,8 +8,11 @@ python:
   - 3.7
 
 env:
-  - KARTOTHEK_ARROW_VERSION=0.12.1
```

Comment:
Instead of relying on the ordering here you should define the environemnt for the OSX job explicitly. I believe you can provide an `env` key which will overwrite this global setting

Reply:
Fair point, changing.